### DISCUSSION TITLE
Is This the Final Form?

### DISCUSSION THREAD

**Discussion by Auri3l** [groepl/Obsidian-Zettelkasten-Starter-Kit#2] [Q&A] [answered]
Hello, thank you for the great work. I just wanted to make sure this is not the final version right? I am currently reading books about zettelkasten (not taking great notes for the time being :)) and wanted to make sure. I want to help.

Best Regards,

**Comment by groepl** [comment, marked as answer]
>> this is not the final version right?

No, it isn't. I'll give you an update for each improval of my Zettelkasten. Also an update to provide some description on how to use the templates. Some hints about tagging and using some templates also for Project Notes.
